codecov.yml

```diff
@@ -9,339 +10,10 @@ comment:
-  easyocr:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/easyocr.py
-  evalml:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/evalml.py
-  fastai:
-    carryforward: true
-    paths:
-      - src/bentoml/fastai.py
-      - src/bentoml/_internal/frameworks/fastai.py
-  fasttext:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/fasttext.py
-  flax:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/flax.py
-  gluon:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/gluon.py
-  h2o:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/h2o.py
-  keras:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/keras.py
-  lightgbm:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/lightgbm.py
-  mlflow:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/mlflow.py
-  onnx:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/onnx.py
-  onnxmlir:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/onnxmlir.py
-  paddle:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/paddle.py
-  picklable_model:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/picklable.py
-  pycaret:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/pycaret.py
-  pyspark:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/pyspark.py
-  pytorch:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/pytorch.py
-  pytorch_lightning:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/pytorch_lightning.py
-  sklearn:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/sklearn.py
-  spacy:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/spacy.py
-  statsmodels:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/statsmodels.py
-  tensorflow:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/tensorflow_v2.py
-  tensorflow_v1:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/tensorflow_v1.py
-  transformers:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/transformers.py
-  xgboost:
-    carryforward: true
-    paths:
-      - src/bentoml/_internal/frameworks/xgboost.py
-  e2e-tests-http:
-    carryforward: true
-    paths:
-      - "src/bentoml/**/*"
-      - "src/bentoml/models.py"
-  e2e-tests-grpc:
-    carryforward: true
-    paths:
-      - "src/bentoml/**/*"
-      - "src/bentoml/grpc/interceptors/"
-      - "src/bentoml/grpc/utils/"
-  unit-tests:
-    carryforward: true
-    paths:
-      - "src/bentoml/**/*"
-      - "src/bentoml/models.py"
+    project: of
```

Comment:
What's this doing?

Reply:
oh i'm just turning off. it is a typo actually.